GiEnJul/Controllers/AdminController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AutoMapper;
+using GiEnJul.Features;
+using Microsoft.AspNetCore.Mvc;
+using Serilog;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GiEnJul.Models;
+namespace GiEnJul.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AdminController : ControllerBase
+    {
+        private readonly IEventRepository _eventRepository;
+        private readonly IGiverRepository _giverRepository;
+        private readonly IRecipientRepository _recipientRepository;
+        private readonly IPersonRepository _personRepository;
+        private readonly ILogger _log;
+        private readonly IMapper _mapper;
+
+        public AdminController(IEventRepository eventRepository, IGiverRepository giverRepository, IRecipientRepository recipientRepository, IPersonRepository personRepository, ILogger log, IMapper mapper)
+        {
+            _eventRepository = eventRepository;
+            _giverRepository = giverRepository;
+            _recipientRepository = recipientRepository;
+            _personRepository = personRepository;
+            _log = log;
+            _mapper = mapper;
+        }
+
+        [HttpGet("querygivers")]
+        public async Task<List<Models.Recipient>> GetUnmatchedRecipientsAsync(string location) {
+            var currentEvent = await _eventRepository.GetActiveEventForLocationAsync(location);
+        return await _recipientRepository.GetUnmatchedRecipientsAsync(location, currentEvent);
+        }
+        
+        [HttpGet("allgivers")]
```

Comment:
Should this path be "givers" instead of "allgivers"? I think this would be better usage of rest api. 

This could probably be applied to "querygivers" as well. Here we could be using e.g. `/recipients?filter=unmatched`

![image](https://user-images.githubusercontent.com/61537815/125039088-69d1b780-e096-11eb-81bb-28bae0dbba05.png)

Reply:
Good suggestion, will implement it!

Reply:
Is it ok if I implement it in another PR?

Reply:
Yeah I think so 👍